bors.toml

```diff
@@ -0,0 +1,10 @@
+# The service providing the commit statuses to GitHub.
+status = ["buildbot/buildbot-build-script"]
```

Comment:
Oops, I told you an untruth earlier.

All of the yorick repos should use:`buildbot/yk-buildbot-build-script-hw`.

Reply:
Truthed in https://github.com/ykjit/ykcpython/pull/2/commits/1fa682352ec60d3a7d45af77e11d6eae907a13ca.

Reply:
Mighty decent of you sir.

Reply:
OK to squash?

Reply:
Hold off for a bit, as I'm fiddling with the CI config at the moment.

We probably also need to update the `yk` git url, as i've moved it into `ykjit`. github might be clever and implement a redirect in the meantime.